src/features/HostQR.js

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import QRCode from 'react-native-qrcode-svg';
```

Comment:
Question: eventually do we need to implement the functionality to generate a new QR code every time 'Create a Session' is clicked? If so, should we create a ticket for finding a QR code generator API?

Reply:
<img width="256" alt="Screen Shot 2021-12-22 at 11 41 31 AM" src="https://user-images.githubusercontent.com/18197013/147146631-76225015-2e3f-4011-95e6-f1445a50082c.png">
I will look more in detail again. My understanding is that you can insert the URL (End Point) in the `value`. It's hard to tell at this point without implementing. We can discuss this more.